public/lib/observe_resize.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function ($elem, fn, frequency) {
```

Comment:
Add comment "Borrowed from `http://github....../observe_resize.js`"

Reply:
I added the reference in a comment at the first line of this file.